packages/puppeteer-core/src/revisions.ts

```diff
@@ -9,5 +9,6 @@
  */
 export const PUPPETEER_REVISIONS = Object.freeze({
   chrome: '121.0.6167.85',
+  'chrome-headless-shell': '121.0.6167.85',
```

Comment:
This need to be kept automatically update, I think we need to update the automation tooling.